static/styles/image_upload_widget.css

```diff
@@ -184,6 +185,12 @@
         top: 30%;
         right: 30%;
     }
+
+    .image_file_input_error {
+        height: 30px;
+        width: 200px;
+        display: block;
```

Comment:
Is this fixed size going to work with other languages that might have a longer translation of the error message?

(I'm wondering if we can avoid needing a `height` field and just having it line-wrap as needed).

Reply:
My bad. I checked with other languages which have longer translations like Spanish and found that they don't fit. and I went with height as I had issues with line-wrapping, like the text was overlapping with the preview organization element. will try to change that.